centos-upgrade-second-stage.sh

```diff
@@ -40,6 +40,11 @@ fi
 echo "Cleaning out old kernels"
 sudo package-cleanup -y --oldkernels --count=1
 
+if test $is_centos6 -eq 1; then
```

Comment:
Did you actually use these changes?  If not, why are they in this commit?

Reply:
I made a note in the second part of the commit message. I did not use it, but tested them outside of the rest of the packer steps and it works with the community AMI; just not when done along with the other updates.  Added it to the packer file so we don't lose it for the next release.

Reply:
ok; fair enough :)